example.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
If it causes tests to fail (haven't investigated yet - just doing the first pass of review now) - you can use pytest's [tmpdir](https://pytest.org/latest/tmpdir.html) module to setup temporary directories & files in order to test resolving relative file paths.